src/Raven.Studio/wwwroot/App/views/database/tasks/ongoingTasks.html

```diff
@@ -272,17 +272,23 @@ <h5 class="tasks-list-item olap-etl no-text-transform">
                                                 <div class="property-name">Task Status:</div>
                                                 <div class="property-value text-details" data-bind="text: badgeText"></div>
                                             </div>
-                                                <!--    TODO ... Destination info is waiting for RavenDB-16435 -->
-<!--                                            <div class="property-item">-->
-<!--                                                <div class="property-name">Destination:</div>-->
-<!--                                                <div class="property-value" title="Destinations that are defined the in connection string">-->
-<!--                                                    <span data-bind="text: destinationDescription"></span>-->
-<!--                                                </div>-->
-<!--                                            </div>-->
+                                            <div class="property-item">
+                                                <div class="property-name">Destination:</div>
```

Comment:
no need to break that in model - you have an access to $index in knockout - hiding this test is view responsibility

Reply:
visible: $index() === 0